# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1513876450 18000
#      Thu Dec 21 12:14:10 2017 -0500
# Node ID 92f0d6013ddd772813b0d3ed816ccdaec85e58c9
# Parent  98a03a1097b45d0419fea3c91934311dc701dbaa
avoid shadowed variable warning from GCC

* graphics.cc (gh_manager::do_execute_callback):
Eliminate duplicate variable.

diff --git a/libinterp/corefcn/graphics.cc b/libinterp/corefcn/graphics.cc
--- a/libinterp/corefcn/graphics.cc
+++ b/libinterp/corefcn/graphics.cc
@@ -10311,22 +10311,20 @@ gh_manager::do_execute_callback (const g
                       << std::endl;
             Flasterr (ovl ("execution error in graphics callback function"));
             octave::interpreter::recover_from_exception ();
           }
 
       // Redraw after interacting with a user-interface (ui*) object.
       if (Vdrawnow_requested)
         {
-          graphics_object go = get_object (h);
-
           if (go)
             {
-              std::string go_name = go.get_properties ()
-                                      .graphics_object_name ();
+              std::string go_name
+                = go.get_properties ().graphics_object_name ();
 
               if (go_name.length () > 1
                   && go_name[0] == 'u' && go_name[1] == 'i')
                 {
                   Fdrawnow ();
                   Vdrawnow_requested = false;
                 }
             }
